lib/rubocop/cop/method_complexity_patch.rb

```diff
@@ -0,0 +1,18 @@
+# Monkey patching! Here be dragons.
+module RuboCop
+  module Cop
+    module MethodComplexity
```

Comment:
I'd define this as:

```
module RuboCop::Cop
  MethodComplexity.module_eval do
    # ...
  end
end
```

Reason: If any of these module names change, it will fail fast.